langchain/scripts/create-entrypoints.js

```diff
@@ -57,6 +57,7 @@ const entrypoints = {
   "vectorstores/hnswlib": "vectorstores/hnswlib",
   "vectorstores/faiss": "vectorstores/faiss",
   "vectorstores/weaviate": "vectorstores/weaviate",
+  "vectorstores/lancedb": "vectorstores/lancedb",
```

Comment:
It does seem like `node langchain/scripts/create-entrypoints.js` has not been executed yet. Would it be possible to run these commands?